give.php

```diff
@@ -267,6 +267,11 @@ public function __wakeup() {
 		 */
 		private function setup_constants() {
 
+			// PHP version
+			if ( ! defined( 'REQUIRED_PHP_VERSION' ) ) {

```

Comment:
Rename this constant to `GIVE_REQUIRED_PHP_VERSION` so it's unique to our plugin